core/src/main/java/hudson/model/Slave.java

```diff
@@ -133,10 +133,7 @@
     private /*almost final*/ DescribableList<NodeProperty<?>,NodePropertyDescriptor> nodeProperties = 
                                     new DescribableList<NodeProperty<?>,NodePropertyDescriptor>(Jenkins.getInstance().getNodesObject());
 
-    /**
-     * Lazily computed set of labels from {@link #label}.
-     */
-    private transient volatile Set<Label> labels;
```

Comment:
Jenkins allow that cloud plugins are able to access to this "private" field using reflection? Otherwise, they can't access to this field.